utilities/methods.py

```diff
@@ -1,4 +1,56 @@
 import requests
+from utilities.configuration import *
+from data.common_headers import *
 
-def create_booking(url, body, header_json):
-    requests.post(url, json=body, headers=header_json, )
+url = main_endpoint + '/booking'
+
+
+def create_session():
+    session = requests.Session()
+    return session
+
+
+def create_url_with_id(booking_id):
+    return main_endpoint + '/booking/' + str(booking_id)
+
+
+def create_auth_url():
+    return main_endpoint + '/auth'
+
+
+def create_booking(booker, session):
```

Comment:
Dzięki metod udało mi się osiągnąć to co chciałam w testach, że są bardziej czyste, ale nadal z nimi kombinuję. Szczególnie nie do końca jestem zadowolona z create_booking i zwracania 4 wartości i potem w teście przypisywania ich do zmiennych i używania w assercji